### ISSUE TITLE
Remove Redundant pages from City-Specific Housing Resources

### ISSUE THREAD

**Issue by clairehubel** [philhawksworth/the-united-effort-orginization#131]
In cases where the link leads to properties that are included in our housing db, I suggest we remove these pages now. 

This prunes out info that is already covered, as well as 'forcing' web traffic through our housing db, which is more complete.

What are your thoughts on this @trevorshannon @philhawksworth ?

**Comment by clairehubel** [comment]
OK, would you like me to work through these and let you know which ones to delete?

**Comment by trevorshannon** [comment]
that would be perfect.

**Comment by clairehubel** [comment]
OK, I worked my way through all of them. The only one that clearly be removed is Los Altos. I tried doing that myself but it didn't work! 

For many of the others, I found a few properties here and there that we need to add, and I will plan to do that in June. Also a bunch of typos that I can now fix.

I tried relabeling the Sunnyvale links, which are both broken, but I must have been doing something wrong. I can send details of this later.

**Comment by trevorshannon** [comment]
Ah, so sorry!  When we moved the housing resources into the Resources section, I moved the underlying housing data from the `Housing` table to the `Resources Database` table.  That's the data used on the website now.  

I have since marked the old `Housing` table as deprecated and moved it away from view.  I copied your edits into the `Resources Database` table. You did everything right!

And just checking: there should be just a single link for Sunnyvale now?
